DemoApp/DemoAppCoordinator+DemoApp.swift

```diff
@@ -0,0 +1,158 @@
+//
+// Copyright © 2022 Stream.io Inc. All rights reserved.
+//
+
+import StreamChat
+import StreamChatUI
+import UIKit
+
+// MARK: - Navigation
+
+extension DemoAppCoordinator {
```

Comment:
Maybe it makes sense to create a **DemoAppShared** folder at the root. to be easier to find the code that is shared with the Test App? 🤔

Reply:
Yes, that makes sense. I've added a Shared folder at the root and organized things a bit better. We can further improve it in the future.